src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @flow
+ * @file Versions Sidebar component
+ * @author Box
+ */
+
+import * as React from 'react';
```

Comment:
Nit. Can we fix order of imports like we decided in the document?
https://confluence.inside-box.net/display/ETO/PR+Conventions

Reply:
@sanjui9000 I think I got it right this time if you want to take a look!